Expensify/App
# Fix back handler for Android

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/4211

### Tests | QA Steps
1. Open LHN on Android.
2. navigate to a report.
3. Use the back button on the Android device.
4. You should land on LHN.
5. Use the back button on the Android device.
6. App should be closed.


### Tested On

- [x] Android (affected)

### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android


https://user-images.githubusercontent.com/24370807/141691522-e19fa22b-62d8-4e96-bbc8-11e78a766fd8.mp4
